install/helm/space-agon/templates/director.yaml

```diff
@@ -32,12 +32,21 @@ spec:
         - name:  {{ template "director.name" .}}
           image: "{{ .Values.director.image.repository }}:{{ .Values.director.image.tag }}"
           imagePullPolicy: {{ .Values.director.imagePullPolicy }}
+          env:
+          - name: OM_CORE_ADDRESS
+            value: {{ .Values.director.omCoreAddress }}
+          - name: MMF_ADDRESS
+            value: {{ .Values.director.mmfAddress }}
+          - name: MMF_PORT
+            value: {{ .Values.director.mmfPort | quote }}
 ---
 apiVersion: v1
 kind: ServiceAccount
 metadata:
   name: {{ .Values.director.serviceAccountName }}
   namespace: {{ .Release.Namespace }}
+  annotations:
+    iam.gke.io/gcp-service-account: om-caller@peteryizhong-gke-dev.iam.gserviceaccount.com
```

Comment:
Could you replace this specific value to helm chart variable and add values.yaml?

For example:
https://github.com/googleforgames/agones/blob/v1.43.0/install/helm/agones/templates/serviceaccounts/controller.yaml#L28

Reply:
Done, I used `YOUR_GOOGLE_SERVICE_ACCOUNT` as a place holder because it looks more intuitive.

Reply:
Thank you. I could confirm `.Values.director.googleServiceAccount`.